typings/jsDocs.ts

```diff
@@ -1056,6 +1056,20 @@ type GroupShout = {
     updated: string;
 }
 
+/**
+ * @typedef
+ */
+type GroupDescriptionResult = {
+    newDescription: string
+}
+
+/**
+ * @typedef
+ */
+type GroupNameResult = {
+    newDescription: string
```

Comment:
newDescription?

Reply:
Oops. Forgot to update JSDOC as well as the typings.